docs/source/narr/aheadofprint.rst

```diff
@@ -46,6 +46,7 @@ os todos os elementos preenchidos: :ref:`elemento-day`, :ref:`elemento-month` e
     </article>
 
 
-.. note::
-	Em AOP considerar sempre a tag :ref:`elemento-month` para indicação de mês. Nunca inserir :ref:`elemento-season`.
+.. note:: Em AOP considerar sempre o elemento :ref:`elemento-month` para indicação de mês. Nunca se deve inserir o elemento :ref:`elemento-season`.
```

Comment:
Inserido como sigla, remetendo a ahead-of-print.

Aqui gostaria de fazer uma sugestão:
- em todos os locais onde devesse apontar para o arquivo aheadofprint.rst, poderia ser grafado como `` :ref:`ahead-of-print` ``;
- para casos em que se deseja que o usuário veja a definição, apontaríamos para a entrada no glossário como `` :term:`Ahead of Print` ``.

O que acham? Se concordarem, preciso apenas de uma lista identificando em que arquivos temos referência a AOP e em cada um como deve ser apontado.